stac_fastapi/pgrest/setup.py

```diff
@@ -0,0 +1,65 @@
+"""arturo-stac-api."""
+import os
+from imp import load_source
+
+from setuptools import find_packages, setup
+
+with open("README.md") as f:
+    desc = f.read()
+
+# Get version from stac-fastapi-api
+__version__ = load_source(
+    "stac_fastapi.api.version",
+    os.path.join(os.path.dirname(__file__), "stac_fastapi/api/version.py"),
+).__version__  # type:ignore
+
+install_requires = [
+    "fastapi",
+    "attrs",
+    "shapely",
+    "pydantic[dotenv]",
+    "stac_pydantic==1.3.8",
+    "titiler",
+]
+
+extra_reqs = {
+    "sqlalchemy": [
+        "sqlakeyset",
+        "geoalchemy2<0.8.0",
+        "sqlalchemy==1.3.23",
+        "shapely",
+        "psycopg2-binary",
+        "alembic",
+        "fastapi-utils",
+    ],
+    "dev": ["pytest", "pytest-cov", "pytest-asyncio", "pre-commit", "requests"],
+    "tiles": ["titiler==0.2.*"],
+    "docs": ["mkdocs", "mkdocs-material", "pdocs"],
+}
+
+
+setup(
+    name="stac-fastapi",
```

Comment:
```suggestion
    name="stac-fastapi.pgrest",
```
?

Reply:
side effect of doing some bulk operations to bootstrap things - pgstac does not even exist in this branch yet